.github/workflows/vm-example.yaml

```diff
@@ -5,6 +5,9 @@ on:
     - cron:  '42 4 * * 2'  # run once a week
   workflow_dispatch: # adds ability to run this manually
 
+env:
+  TARGET_ARCH: amd64
```

Comment:
Is it worth adding this as a workflow parameter, to allow running for ARM? No pressure to do so in this PR — always possible to make the change later if not immediately useful.

Reply:
I have a separate issue to get everything working in CI #1089 (this one more dedicated to actual builds) and #1084 (this one more dedicated to have working e2e tests in CI)